data_steward/tools/generate_ext_tables.py

```diff
@@ -5,11 +5,10 @@
 import constants.bq_utils as bq_consts
 import constants.cdr_cleaner.clean_cdr as cdr_consts
 from resources import fields_for, MAPPING_TABLES
+from common import JINJA_ENV
 
 LOGGER = logging.getLogger("__name__")
```

Comment:
This needs to be removed.  This is setting the logger name to the string __name__.  So you would see logs like this.
```
2020-11-03 12:18:03,439 - INFO - __name__ - using dynamic extension table schema for table: procedure_occurrence_ext
```